driver-core/src/main/java/com/datastax/driver/core/Metadata.java

```diff
@@ -116,7 +116,7 @@ static String handleId(String id) {
             return id.toLowerCase();
 
         // Check if it's enclosed in quotes. If it is, remove them and unescape internal double quotes
-        if (id.charAt(0) == '"' && id.charAt(id.length() - 1) == '"')
+        if (!id.isEmpty() && id.charAt(0) == '"' && id.charAt(id.length() - 1) == '"')
```

Comment:
Noticed a `StringIndexOutOfBoundsException` being raised here if the identifier is the empty string. With the fix for JAVA-993, empty identifiers shouldn't arrive anymore, but I think it doesn't hurt to check the string length here.